tests/cases/unittests/session.ts

```diff
@@ -0,0 +1,454 @@
+/// <reference path="..\..\..\src\harness\harness.ts" />
+
+module ts.server {

```

Comment:
Noting that we should add a tslint rule to prefer the new `namespace` keyword over `module`.